cypress/e2e/specialFeatures/atiAnalytics/assertions/featuresAnalysis.js

```diff
@@ -38,10 +38,7 @@ export const assertFeaturesAnalysisComponentClick = ({
       cy.get('[data-testid="features"]').scrollIntoView({ duration: 1000 });
 
       // Click on first item
-      cy.get('[data-testid="features"]')
-        .find('a')
-        .first()
-        .click({ force: true });
+      cy.get('[data-testid="features"]').find('a').first().click();
```

Comment:
I am wondering if we should revert this change, given that it's not related to the audio / podcast page tests?

Reply:
(My apologies, I requested that these changes were pulled in from another PR (#12423), not realising it included unrelated changes).

Reply:
no worries ! i should have caught that they weren't very related myself so no stress, i agree and will remove now